dependent/processing-and-application-of-ceramics.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+  <info>
+    <title>Processing and Application of Ceramics</title>
+    <id>http://www.zotero.org/styles/processing-and-application-of-ceramics</id>
+    <link href="http://www.zotero.org/styles/processing-and-application-of-ceramics" rel="self"/>
+    <link href="http://www.zotero.org/styles/frontiers-in-optics" rel="template"/>
+    <link href="http://www.tf.uns.ac.rs/publikacije/PAC/index.html" rel="documentation"/>
+    <author>
+      <name>Santiago Maya-Johnson</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <eissn>2406-1034</eissn>
+    <updated>2019-01-02T12:24:45+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="presented at">presented at</term>
+    </terms>
+  </locale>
```

Comment:
This whole locale section can be deleted as it's not doing anything. (probably just something you didn't change from the template style)